formula_renames.json

```diff
@@ -20,6 +20,7 @@
   "cassandra22": "cassandra@2.2",
   "cdiff": "ydiff",
   "cloog018": "cloog",
+  "cluster-api": "clusterctl",
```

Comment:
I think we can skip this, as the formula is just 18 hours old. It can't have picked up too many downloads since then.